rc/tools/make.kak

```diff
@@ -51,25 +53,27 @@ define-command -hidden make-open-error -params 4 %{
 }
 
 define-command -hidden make-jump %{
-    evaluate-commands %{
+    evaluate-commands -save-regs / %{
+        set-register / %sh{ if [ -z "$kak_opt_make_error_line_pattern" ] ; then printf %s "^(?:\w:)?[^:\n]+:\d+:(?:\d+:)?$kak_opt_make_error_pattern" ; else printf %s "$kak_opt_make_error_line_pattern"; fi }
```

Comment:
this is only needed in the `catch` block, move it there to reduce scope?

Reply:
done